packages/libxfce4ui.rb

```diff
@@ -0,0 +1,23 @@
+require 'package'
+
+class Libxfce4ui < Package
+    description 'Replacement of the old libxfcegui4 library'
+    homepage "https://xfce.org/"
+    version '4.15.3'
+    compatibility 'all'
+    source_url "https://archive.xfce.org/src/xfce/libxfce4ui/4.15/libxfce4ui-#{version}.tar.bz2"
```

Comment:
I prefer to not use `#{version}` in urls.  It makes it easier to `sha256sum` and cut and paste.